addons/controllers/csi/vspherecsiconfig_utils.go

```diff
@@ -360,3 +367,25 @@ func (r *VSphereCSIConfigReconciler) getNumberOfControlPlaneNodes(ctx context.Co
 	}
 	return *kcp.Spec.Replicas, nil
 }
+
+// Return true if Valid Availability Zones are present
+func (r *VSphereCSIConfigReconciler) HasValidAZ(ctx context.Context) (bool, error) {
```

Comment:
Can this be renamed to may be isStretchedSupervisorCluster or something similar as this is being used to determine that?